src/widget/wtracktableview.cpp

```diff
@@ -590,44 +593,48 @@ void WTrackTableView::mouseMoveEvent(QMouseEvent* pEvent) {
             if (!index.isValid()) {
                 continue;
             }
-            locations.append(trackModel->getTrackLocation(index));
+            locations.append(pTrackModel->getTrackLocation(index));
         }
         DragAndDropHelper::dragTrackLocations(locations, this, "library");
     }
 }
 
 // Drag enter event, happens when a dragged item hovers over the track table view
 void WTrackTableView::dragEnterEvent(QDragEnterEvent * event) {
-    auto* trackModel = getTrackModel();
+    auto* pTrackModel = getTrackModel();
+    if (!pTrackModel || !event->mimeData()->hasUrls()) {
+        event->ignore();
+    }
```

Comment:
```suggestion
        event->ignore();
        return;
    }
```

Reply:
Thanks! I fixed this also in other places.